tests/test_helper/bats/cluster.bats

```diff
@@ -46,3 +46,115 @@ setup() {
         assert [ "$test_family" = "$addr_family" ]
     done
 }
+
+function _test_db_clustered() {
+    local nbsb=$1; shift
+
+    for container in $TEST_CONTAINERS; do
+        local container_services
+        container_services=$(microovn_get_cluster_services "$container")
+        if [[ "$container_services" != *"central"* ]]; then
+            echo "Skip $container, no central services" >&3
+            continue
+        fi
+
+        echo "Checking DB clustered from ${container}'s point of view" >&3
+        local expected_addr
+        expected_addr=$(test_is_ipv6_test && echo "[" || true)
+        expected_addr+=$(microovn_get_cluster_address "$container")
+        expected_addr+=$(test_is_ipv6_test && echo "]" || true)
+        local expected_port
+        [ "$nbsb" == "nb" ] && expected_port=6643 || expected_port=6644
+
+        run microovn_ovndb_cluster_status "$container" "$nbsb"
+
+        assert_success
+        assert_line "Address: ssl:${expected_addr}:${expected_port}"
+        assert_line "Status: cluster member"
+    done
+}
+
+@test "OVN Northbound DB clustered" {
+    _test_db_clustered nb
+}
+
+@test "OVN Southbound DB clustered" {
+    _test_db_clustered sb
+}
+
+function _get_member_cluster_address() {
+    # build a list of cluster addresses by interrogating every member
+    #
+    # Note that we do it this way intentionally to help uncover any
+    # inconsistencies (as opposed to asking a single member for all the
+    # addresses).
+    for container in $TEST_CONTAINERS; do
+        container_services=$(microovn_get_cluster_services "$container")
+        if [[ "$container_services" == *"central"* ]]; then
+            echo "$(microovn_get_cluster_address $container)"
+        fi
+    done
+}
+
+@test "Chassis Open_vSwitch external_ids:ovn-remote addresses" {
+    local cluster_addresses=()
+    local container_services
+
+    readarray -t cluster_addresses < <(_get_member_cluster_address)
+    assert_equal "${#cluster_addresses[@]}" 3
+    for container in $TEST_CONTAINERS; do
+        container_services=$(microovn_get_cluster_services "$container")
+        if [[ "$container_services" != *"chassis"* ]]; then
+            echo "Skip $container, no chassis services" >&3
+            continue
+        fi
+
+        run lxc_exec \
+            "$container" \
+            "microovn.ovs-vsctl get Open_vSwitch . external_ids:ovn-remote"
+        for addr in "${cluster_addresses[@]}"; do
+            expected_addr=$(test_is_ipv6_test && echo "[" || true)
+            expected_addr+=$addr
+            expected_addr+=$(test_is_ipv6_test && echo "]" || true)
+            assert_output -p "$expected_addr"
```

Comment:
I'd be weary of partial matching especially wrt IP addresses. If output contains
```
10.0.0.100
10.0.0.101
```
and we are looking for `10.0.0.10` this assertion would pass

Reply:
That is a good point, I was hoping to evade the need for parsing `external_ids:ovn-remote`, but I guess we have to due to this concern, will do and post an update. Thank you for pointing it out!

Reply:
Looking at this again, I still think full parsing here, and the extra looping required to make use of it would not be worth the effort. Making use of a fully qualified search string (i.e. `ssl:$expected_addr:6642`) should be more than good enough to avoid false matches.